publication/ver11/societal_impact.md

```diff
@@ -0,0 +1,239 @@
+
+# Results of Societal Impact self-review for Web of Things (WoT) 
+
+This document was prepared by the Web of Things (WoT) Working Group,
+responding to the questions raised in the February 9, 2023 version of the 
+[Societal Impact Questionnaire](https://www.w3.org/2001/tag/doc/draft-finding-societal-impact-questionnaire-20230209/) document published by the W3C and required as part of the 
+wide review process.
+
+These responses relate to all WoT deliverables being developed as part of the current charter.
```

Comment:
```suggestion
These responses relate to all normative WoT deliverables being developed as part of the current charter.
```

Reply:
I think the comments apply to more than the normative deliverables.  In particular, security and privacy guidelines is an informative document and is relevant.  So I'd like to leave this as is.

Reply:
However, I see security, scripting, and use cases were not mentioned, so I will add them to the list of documents.

Reply:
I would not overcomplicate here. The normative deliverables do references/pointers to our Note documents.